nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/CreateSystemProperties.java

```diff
@@ -154,6 +157,17 @@ public void execute(AdminCommandContext context) {
                 } finally {
                     TranslatedConfigView.doSubstitution.set(Boolean.TRUE);
                 }
+
+                String HTML_JS_PATTERN = "<[^>]+>";
+                Pattern pattern = Pattern.compile(HTML_JS_PATTERN);
```

Comment:
The Pattern and its String can be made static to avoid compilation on every execution.